network-framework/scripts/invoke.sh

```diff
@@ -74,18 +74,33 @@ parseParameters() {
   PARAMS="$PARAMS]}"
 }
 
+# Check if invocation is run in quiet mode
+isQuiet=0
+while getopts "q" opt; do
+  case "$opt" in
+  q)
+    isQuiet=1
+    shift
+    ;;
+   esac
+done
 
 parseParameters $@
 res=$?
 verifyResult $res "Invoke transaction failed on channel '$CHANNEL_NAME' due to uneven number of peer and org parameters
 or lack of function name"
-set -x
-echo $PARAMS
-peer chaincode invoke -o orderer.example.com:7050 --tls $CORE_PEER_TLS_ENABLED --cafile $ORDERER_CA -C $CHANNEL_NAME -n mycc $HARD_PEER_CONN_PARMS -c $PARAMS >&log.txt
-#peer chaincode invoke -o orderer.example.com:7050 --tls $CORE_PEER_TLS_ENABLED --cafile $ORDERER_CA -C $CHANNEL_NAME -n mycc $PEER_CONN_PARMS -c $PARAMS >&log.txt
-res=$?
-set +x
-cat log.txt
-verifyResult $res "Invoke execution on $PEERS failed "
-echo "===================== Invoke transaction successful on $PEERS on channel '$CHANNEL_NAME' ===================== "
-echo
\ No newline at end of file
+if [ $isQuiet -eq 1 ]; then
+  peer chaincode invoke -o orderer.example.com:7050 --tls $CORE_PEER_TLS_ENABLED --cafile $ORDERER_CA -C $CHANNEL_NAME -n mycc $HARD_PEER_CONN_PARMS -c $PARAMS >&log.txt
+  res=$?
+  verifyResult $res "Invoke execution on $PEERS failed "
```

Comment:
In quiet mode, please translate failures to an exit status of 1 (instead of 0).

Reply:
In case of failure, verifyResult will exit with status 1.

Reply:
I see, okay. Carry on, then.